go.mod

```diff
@@ -67,15 +67,15 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/zalando/go-keyring v0.2.3
 	go.uber.org/atomic v1.11.0
-	golang.org/x/crypto v0.21.0
-	golang.org/x/sync v0.6.0
+	golang.org/x/crypto v0.23.0
+	golang.org/x/sync v0.7.0
 	golang.org/x/sys v0.20.0
-	golang.org/x/term v0.18.0
+	golang.org/x/term v0.20.0
 	golang.org/x/tools v0.17.0
 	google.golang.org/genproto v0.0.0-20240205150955-31a09d347014
-	google.golang.org/grpc v1.61.0
+	google.golang.org/grpc v1.64.0
```

Comment:
Did we mean to update to 1.64? I thought we wanted 1.61.1?

Reply:
Oh that's annoying.. I put in 1.61.1 but when I run go mod tidy it turns it to 1.64.0 :<

Reply:
If you use `go get google.golang.org/grpc@v1.61.1`, does it work?